packages/override-tools/src/Cli.ts

```diff
@@ -179,6 +207,96 @@ async function removeOverride(overridePath: string) {
   console.log(chalk.greenBright('Override successfully removed'));
 }
 
+/**
+ * Attempts to automatically merge changes from the current version into
+ * out-of-date overrides.
+ */
+async function autoUpgrade(manifestPath: string, version?: string) {
+  return doUpgrade(manifestPath, false /*isManual*/, version);
+}
+
+/**
+ * Stages the base versions of all overrides before writing conflict-marker
```

Comment:
Need to update this documentation. We used to stage base files before generating conflict markers but I decided to remove that.